spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'post_show', type: :feature do
```

Comment:
- [ ] Kindly check on your test. I run into the error below while running your tests.
![image](https://user-images.githubusercontent.com/61408860/192567799-3bf60d2d-ab68-45f2-a148-a7ada9d1db88.png)